utils/is/src/is.mjs

```diff
@@ -0,0 +1,142 @@
+
+// @todo add a way to set the config
+const config = {}
+
+////
+/// @name is
+/// @author Tyler Benton
+////
+
+export const is = {
+  /// @name is.env
+  /// @description This function is used to check against the current env
+  /// @arg {string, array} ...envs - The environments to test
+  /// @returns {boolean} if any of the envs passed in match the current
+  /// environment then it will return true
+  /// @example
+  /// is.env('development')
+  env (...envs) {
+    const nodeEnv = process.env.NODE_ENV
+    for (const env of [].concat(...envs)) {
+      // return true if any of them match
+      // react doesn't set the nodeEnv to be `live`, it uses `production`
+      // so we have to test `live` differently
+      if ([ 'storybook', 'test', 'development' ].includes(env) && nodeEnv === env) return true
+      if (config.env === env) return true
+    }
+
+    return false
+  },
+
+  json (obj) {
+    if (!is.string(obj)) return false
+
+    try {
+      JSON.parse(obj)
+      return true
+    } catch (e) {
+      return false
+    }
+  },
+
+  /// @name type
+  /// @description This will check against multiple types
+  /// @alias is.any
+  /// @arg {*} obj - The item to test against
+  /// @arg {array, string} - The types to test against
+  /// @returns {boolean}
+  type (obj, types) {
+    const objType = Object.prototype.toString
+      .call(obj)
+      .slice(8, -1)
+      .toLowerCase()
+    return [].concat(types).includes(objType)
+  },
+
+  /// @name falsy
+  /// @descriptions Determines if something is falsy
+  /// @arg {*} obj - The item to test against
+  /// @returns {boolean}
+  falsy (obj) {
+    return !obj
+  },
+
+  /// @name truthy
+  /// @descriptions Determines if something is truthy
+  /// @arg {*} obj - The item to test against
+  /// @returns {boolean}
+  truthy (obj) {
+    return !!obj
+  },
+
+  /// @name null
+  /// @descriptions Determins if something is null or undefined
+  /// @arg {*} obj - The item to test against
+  /// @returns {boolean}
+  null (obj) {
```

Comment:
I feel like you could just check === null without using `is` at all in that case

Reply:
`null` should always return `true` for `undefined` because it's doing a check for `something == null`. Since we really don't ever need to tell the difference between `null` and `undefined` we can do what william said and just check `something === null`